app/components/TaskListItem.tsx

```diff
@@ -0,0 +1,129 @@
+import { faCaretDown, faCaretUp } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Link from "next/link";
+import { useState } from "react";
+import TaskListStatus from "./TaskListStatus";
+
+interface Props {
+  defaultExpandedState: boolean;
+  listItemNumber: string;
+  listItemName: string;
+  listItemMode?: string;
+  formListItemArray: Array<{
+    stepName: string;
+    linkUrl: { pathname: string; query: { projectRevision: string } };
+    formTitle: string;
+    formStatus: string | JSX.Element;
+  }>;
+  currentStep: string;
+  mode: string;
+}
+
+const TaskListItem: React.FC<Props> = ({
+  defaultExpandedState,
+  listItemNumber,
+  listItemName,
+  listItemMode,
+  formListItemArray,
+  currentStep,
+  mode,
+}) => {
+  const [isExpanded, setIsExpanded] = useState(defaultExpandedState);
+  const toggleCustomAccordion = () => setIsExpanded(!isExpanded);
+
+  const getFormListItem = ({
+    index,
+    stepName,
+    linkUrl,
+    formTitle,
+    formStatus,
+  }) => {
+    return (
+      <li
+        aria-current={currentStep === stepName ? "step" : false}
+        className="bordered"
```

Comment:
is this class name having any effect?

Reply:
Actually, I didn't touch that one but yeah it adds borders at the bottom of each task list item.
![Screen Shot 2022-05-10 at 4 01 51 PM](https://user-images.githubusercontent.com/52381975/167738592-3522b6ec-4736-42e4-a8d7-ea9d12b4fbb4.png)